### PR TITLE
Show usermaterials on the dashboard

### LINKED ISSUE #2230: Provide "LM Agenda View" on Student Dashboard
Allow for a weekly type view of current resources as a parallel to our Agenda view in the dashboard -- instead of events, it would list the current and upcoming list of LMs associated with the activities in the Agenda list, sorted by week/due date.

udpated

this is in conjunction with #1665

### PR DESCRIPTION
**Pull request by jrjohnson**
Allows the user to look at materials upcoming in the next 60 days.

WIP:
- [x] Translations
- [x] dashboard-materials test
- [x] Cleanup tests for new selection of calendar

Fixes #2230

### CODE REVIEW

**Comment by jrjohnson** [comment]
Deploy preview ready!

Built with commit 2d0858136f747cc35bebf110773a10e5224c7e57

https://deploy-preview-2375--ilios-frontend.netlify.com

**Comment by jrjohnson** [comment]
@saschaben this is ready for review.

**Comment by saschaben** [comment]
NICE!!!!! :+1:

File: app/components/dashboard-materials.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+  materials: computed('currentUser.currentUserId', function() {
+    const from = moment().hour(0).minute(0).unix();
+    const to = moment().hour(23).minute(59).add(this.daysInAdvance, 'days').unix();
+    const ajax = this.get('ajax');
+    const host = this.get('host');
+    const currentUser = this.get('currentUser');
+    const namespace = this.get('namespace');
+
+    return new Promise(resolve => {
```
Comment (stopfstedt) [inline app/components/dashboard-materials.js]: Promise needs to be imported.

Comment (jrjohnson) [inline app/components/dashboard-materials.js]: Good catch.  Fixed.

**Pull request merged**